test/parallel/test-quic-client-server.js

```diff
@@ -143,7 +143,7 @@ server.on('session', common.mustCall((session) => {
     // Should not error and should return true... also shouldn't
     // cause anything else to fail.
 
-    assert(session.updateKey());
+    setTimeout(() => assert(session.updateKey()), 500);
```

Comment:
Why the timeout? It sounds like something that could make this test flaky?

Reply:
this is temporary for now. With the update to ngtcp2, we can't perform the key update at this point in the lifecycle because we're still inside the ngtcp2 callback at this point. I'll be removing the key update from this test and moving it into it's own test soonish

Reply:
I think I also remember this test failing because the session wasn’t alive at this point anymore?